packages/graphql-proxy-server/src/server-config/server-config.ts

```diff
@@ -0,0 +1,36 @@
+type SentryConfig = {
+  sentryDsn?: string; // process.env.GRAPHQL_PROXY_SENTRY_DSN
+  sentryEnvironment?: string; // process.env.GRAPHQL_PROXY_SENTRY_ENVIRONMENT
+};
+
+type NodeConfig = {
+  env?: string;
```

Comment:
Shouldn't this be nodeEnv instead? https://www.apollographql.com/docs/apollo-server/api/apollo-server/#nodeenv

Reply:
Nevermind. Sorry, I didn't notice the express-implementation.